test/known_issues/test-shadow-realm-gc.js

```diff
@@ -8,5 +8,6 @@
 require('../common');
 
 for (let i = 0; i < 1000; i++) {
-  new ShadowRealm();
+  const realm = new ShadowRealm();
+  realm.evaluate(`new TextEncoder(); 1;`);
```

Comment:
Since the case is failing with this, we should mark it as `SKIP` in the `known_issues.status` as well.